src/pages/SpeedMarkets/SpeedMarkets.tsx

```diff
@@ -400,6 +401,13 @@ const SpeedMarkets: React.FC = () => {
     );
 };
 
+const getTimeStampForDelta = (seconds: number) => {
```

Comment:
Returns undefined for parameter zero, not sure if that is intention.
Function used in one place, it can be replaced with inline function from date-fns: `addSeconds(Date.now(), deltaTimeSec)`

Reply:
Yep, its intended to return undefined when expiration date was selected. I would leave it for this iteration, as there are some more refactoring and improvements to do on the chart in the future.
Improvements:
- Add area for range markets
- Dynamically add data to the chart as users is scrolling into the past